detectletters.c

```diff
@@ -0,0 +1,38 @@
+//Olivier Gabison - Tells you the frequency of letters in a string.
+//NOTE: I don't understand the error it's giving me.
+
+#import <stdio.h>
+
+int main(){
+
+  printf("Please input a word\n");
+
+  char word[100];
+  char letters[100];
+  int values[100];
+
+  scanf("\n%s",&word);
```

Comment:
So your warning can be solved by omitting the ampersand. Remember that the parameter for scanf() must be of type ```char*```